core/src/avm2/globals/flash/desktop/NativeApplication.as

```diff
@@ -0,0 +1,259 @@
+
+    public function get startAtLogin():Boolean
+    {
+      stub_getter("flash.desktop.NativeApplication", "startAtLogin");
+      return false;
+    }
+
+    public function set startAtLogin(param1:Boolean):void
+    {
+      stub_setter("flash.desktop.NativeApplication", "startAtLogin");
+    }
+
+    public function activate(window:NativeWindow = null):void
+    {
+      stub_method("flash.desktop.NativeApplication", "activate");
+    }
+
+    public function get activeWindow():NativeWindow
+    {
+      stub_getter("flash.desktop.NativeApplication", "activeWindow");
+      return _openedWindows[0];
+    }
+
+    public function get openedWindows():Array
+    {
+      stub_getter("flash.desktop.NativeApplication", "openedWindows");
+      return _openedWindows;
+    }
+
+    public function get timeSinceLastUserInput():int
+    {
+      stub_getter("flash.desktop.NativeApplication", "timeSinceLastUserInput");
+      return 100;
+    }
+
+    public function get idleThreshold():int
+    {
+      return this._idleThreshold;
+    }
+
+    public function set idleThreshold(value:int):void
+    {
+      this._idleThreshold = value;
+    }
+
+    public function copy():Boolean
+    {
+      stub_getter("flash.desktop.NativeApplication", "copy");
+      return false;
+    }
+
+    public function cut():Boolean
+    {
+      stub_getter("flash.desktop.NativeApplication", "cut");
+      return false;
+    }
+
+    public function paste():Boolean
+    {
+      stub_getter("flash.desktop.NativeApplication", "paste");
+      return false;
+    }
+
+    public function clear():Boolean
+    {
+      stub_getter("flash.desktop.NativeApplication", "clear");
+      return false;
+    }
+
+    public function selectAll():Boolean
+    {
+      stub_method("flash.desktop.NativeApplication", "selectAll");
+      return false;
+    }
+
+    public function getDefaultApplication(extension:String):String
+    {
+      stub_method("flash.desktop.NativeApplication", "getDefaultApplication");
+      return null;
+    }
+
+    public function isSetAsDefaultApplication(extension:String):Boolean
+    {
+      stub_method("flash.desktop.NativeApplication", "isSetAsDefaultApplication");
+      return true;
+    }
+
+    public function setAsDefaultApplication(extension:String):void
+    {
+      stub_method("flash.desktop.NativeApplication", "setAsDefaultApplication");
+    }
+
+    public function removeAsDefaultApplication(extension:String):void
+    {
+      stub_method("flash.desktop.NativeApplication", "removeAsDefaultApplication");
+    }
+
+    [API("681")]
+    public function get executeInBackground():Boolean
+    {
+      stub_getter("flash.desktop.NativeApplication", "executeInBackground");
+      return false;
+    }
+
+    [API("681")]
+    public function set executeInBackground(param1:Boolean):void
+    {
+      stub_setter("flash.desktop.NativeApplication", "executeInBackground");
+    }
+
+    // [API("721")] Ruffle doesn't support this API Version
```

Comment:
I think it would be better to still use an `API` annotation, with the highest supported value; but also leaving the correct one as a comment.